src/coprocessor/tracker.rs

```diff
@@ -163,6 +165,10 @@ impl Tracker {
         self.total_storage_stats.add(&storage_stats);
     }
 
+    pub fn collect_kv_read_time(&mut self, exec_summary: ExecSummary) {
+        self.kv_engine_read_time_ms = exec_summary.time_processed_ns as u64;
```

Comment:
emm

Reply:
Oh, I forgot it ! Will rename it later :)